Tests/Unit/InMemoryRefreshTokenManagerTest.php

```diff
@@ -21,47 +21,109 @@ public function testClearExpired(): void
         timecop_freeze(new DateTimeImmutable());
 
         try {
-            $testData = $this->buildClearExpiredTestData();
+            $testData = $this->buildTestData(
```

Comment:
@franjo-zadelj-trikoder This still uses the `buildTestData` stuff which we've agreed that should be removed, but it is still present here.

Reply:
explained above